app/src/main/java/com/simon/harmonichackernews/CommentsFragment.java

```diff
@@ -564,7 +564,11 @@ public WindowInsetsCompat onApplyWindowInsets(@NonNull View v, @NonNull WindowIn
                 Insets insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemBars());
 
                 float offset = BottomSheetBehavior.from(bottomSheet).calculateSlideOffset();
-                recyclerView.setTranslationY(-insets.top * (1 - offset));
+                float oldY = recyclerView.getTranslationY();
+                float newY = -insets.top * (1 - offset);
+                if (oldY != newY) {
```

Comment:
Btw I don't think this check adds anything because the view already does the same check https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/view/View.java;drc=50c590e82c167a07f7fae84d81293fa52aad93af;l=18430

Reply:
I assumed so but added this because there was an animation glitch on tablets and at the time was under the impression that this fixed it (which I thought was weird). However now I removed the check and things still look the same so I guess it was something else. The check is now removed.